ios/FluentUI/Command Bar/CommandBarAppearance.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public struct CommandBarButtonAppearance {
+    public var tintColor: UIColor
+    public var backgroundColor: UIColor
+    public var highlightedTintColor: UIColor
+    public var highlightedBackgroundColor: UIColor
+
+    public init(
+        tintColor: UIColor = Defaults.tintColor,
+        backgroundColor: UIColor = Defaults.backgroundColor,
+        highlightedTintColor: UIColor = Defaults.highlightedTintColor,
+        highlightedBackgroundColor: UIColor = Defaults.highlightedBackgroundColor
+    ) {
+        self.tintColor = tintColor
+        self.backgroundColor = backgroundColor
+        self.highlightedTintColor = highlightedTintColor
+        self.highlightedBackgroundColor = highlightedBackgroundColor
+    }
+
+    public enum Defaults {
+        public static let tintColor: UIColor = Colors.gray400
+        public static let backgroundColor: UIColor = Colors.surfaceSecondary
+        public static let highlightedTintColor = UIColor(light: Colors.communicationBlue, dark: .black)
+        public static let highlightedBackgroundColor = UIColor(light: UIColor(red: 224 / 255, green: 238 / 255, blue: 249 / 255, alpha: 1), dark: Colors.communicationBlue)
```

Comment:
what's this color?

Reply:
I think this should be msfcolors.primarytint30

Reply:
These colors are default colors. They can be configured via `CommandBarAppearance`. Should we use a fixed `MSFColor` or just replace the defaults?

Reply:
> what's this color?

This is the background color for selected buttons.

![Screen Shot 2020-12-17 at 13 20 57](https://user-images.githubusercontent.com/12813829/102447301-b4440600-406a-11eb-8aed-fd167b0ec33b.png)